### PR TITLE
Add documentation for the --name-print-revision option.

### PR DESCRIPTION
**Pull request by xorrkaz**
Looks like my edit also pruned some trailing whitespace...

### CODE REVIEW

File: pyang/plugins/tree.py [hunk 1]
```diff
@@ -403,6 +407,22 @@ def get_typename(s):
                 return "-> %s" % "/".join(target)
             else:
                 return t.arg
+        elif verbose_types and t.arg == 'enumeration':
+            ret = t.arg
+            es = t.search('enum')
+            if es is not None:
```
Comment (mbj4668) [inline pyang/plugins/tree.py]: search() returns a list of statements, and if there is no 'enum' statement this list will be empty.  So 'if es is not None' will always be true.

Comment (mbj4668) [reply pyang/plugins/tree.py]: Did you consider using 'width' in the formatting of the enumerations?  Maybe create the string and if it is too wide fallback to the old format?  Or split into multiple lines.

**Comment by xorrkaz** [comment]
Ugh.  Thanks, Martin.  But this patch shouldn't have been part of the PR.  I just wanted this PR to focus on the man page documentation for the name.py changes.  But I appreciate the comments.  I will make changes in my tree.

**Comment by mbj4668** [comment]
Also note that the real source for the documentation is in the 'doc' directory.  You should update the pyang.1.dbk file.   Then, for legacy reasons, the nroff file is also checked in.  This is unfortunate, and not even documented.  I should add this to the README file or something...

**Comment by xorrkaz** [comment]
Thanks.  I've updated the docbook docs.

**Comment by xorrkaz** [comment]
Closing to replace with a cleaner one.

**Pull request closed without merging**